AR-builder.groovy

```diff
@@ -308,208 +308,208 @@ pipeline {
 
             }
         }
-
```

Comment:
We can keep it as-is this Jenkins file, without disabling anything, we want to make sure that we have a full working version in GH actions before touching this :D

Reply:
yeah leftover from some tests